pymedphys/_dicom/anonymise/__init__.py

```diff
@@ -133,6 +175,27 @@ def anonymise_dataset(  # pylint: disable = inconsistent-return-statements
         An anonymised version of the input DICOM dataset.
     """
 
+    if replacement_strategy is not None:
```

Comment:
I believe now this will never be None?

Reply:
I think it is None when invoked from DicomBase and tested in test_base